script/debug

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+sbt -jvm-debug 5005 run
```

Comment:
not super wedded to this script, opinions welcome

Reply:
Could we have only one script that takes a `debug` parameter?

Reply:
Updated.